.azure/templates/jobs/build/release_artifacts.yaml

```diff
@@ -11,13 +11,21 @@ jobs:
       # Install Prerequisites
       - template: "../../steps/prerequisites/install_yq.yaml"
       - template: "../../steps/prerequisites/install_asciidoctor.yaml"
+      - template: '../../steps/prerequisites/install_java.yaml'
 
       # Build the release files
       - bash: "make release"
         displayName: "Build release files"
         env:
           RELEASE_VERSION: '${{ parameters.releaseVersion }}'
 
+      # Build Java code with changed release version
+      - bash: "make java_install"
+        displayName: "Build Java code"
+        env:
+          MVN_ARGS: "-DskipTests -e -V -B"
+          CURL_ARGS: "-s"
```

Comment:
What does it need `curl` for?

Reply:
Copy paste from different pipeline, I will remove it, thanks.